### PR TITLE
update docs & README.md

### PR DESCRIPTION
**Pull request by kamo-naoyuki**
- Moved some contents from README to docs.README.md should be more simplified.
   - I left attractive information in README.md i.e. About key feature and demo
- I also moved the contents from docker/README.md to docs @Fhrozen Please check.  
   - I'd like to unify all information to docs. 
- I deleted `Step 2-B) installation including Kaldi installation` from installation documents  and added description about kaldi installation.

### CODE REVIEW

**Comment by Fhrozen** [comment]
LGTM, the docker part.

File: README.md [hunk 1]
```diff
@@ -90,329 +58,15 @@ and also follows [Kaldi](http://kaldi-asr.org/) style data processing, feature e
...
-$ pip install pip --upgrade; pip uninstall matplotlib; pip --no-cache-dir install matplotlib
-```
-
+## Usage
+See https://espnet.github.io/espnet/tutorial.html
 
 ## Docker Container
 
-go to docker/ and follow [README.md](https://github.com/espnet/espnet/tree/master/docker/README.md) instructions there.
+go to docker/ and follow https://espnet.github.io/espnet/docker.html instructions there.
```
Comment (Fhrozen) [inline README.md]: should not this line need to be changed to:
`go to doc/ and follow [instructions](https://espnet.github.io/espnet/docker.html) there.`

Comment (kamo-naoyuki) [reply README.md]: I think it should point to docs because all contents was moved. Why do you think so?

Comment (Fhrozen) [reply README.md]: My only concern is that the links is showed.  So try to hide it with:  `Follow [instructions](https://espnet.github.io/espnet/docker.html) in docs.` then?

**Comment by sw005320** [comment]
LGTM.

File: README.md (hunk 1 above)
Comment (kamo-naoyuki) [reply README.md]: o.k. Thanks!

**Pull request merged**